devtools/platform-descriptor-json/src/test/resources/fakeextensions.json

```diff
@@ -1000,6 +1000,19 @@
             "version": "999-SNAPSHOT",
             "description": "Use Spring Data JPA annotations to create your data access layer"
         },
+        {
+            "name": "Quarkus Extension for Spring Boot properties",
+            "metadata": {
+                "keywords": [
+                    "spring-boot",
+                    "properties"
+                ]
+            },
+            "group-id": "io.quarkus",
+            "artifact-id": "quarkus-spring-boot-properties",
+            "version": "999-SNAPSHOT",
+            "description": "Use Spring Boot properties annotations to configure our application"
+        },
```

Comment:
I don't think you need that.

Reply:
OK I'll remove it. I saw other spring modules mentioned in this file so I followed the trend.

Reply:
I used to think the same :). Let's keep the streams in the other uses since those are for new classes